src/typescript/6-3-class-constructor-initialized.ts

```diff
@@ -1,6 +1,6 @@
 class Dinosaur {
   name: string;
-  age: number = 0;
+  age = 0; // type number is inferred
```

Comment:
have we already explained "inferring" by this point in the training?

Reply:
yes! I will remove the comment.